client/data/transactions/hooks.ts

```diff
@@ -37,6 +37,8 @@ export interface Transaction {
 	transaction_id: string;
 	date: string;
 	type: 'charge' | 'refund' | 'financing_payout' | 'financing_paydown';
+	channel: 'in_person' | 'online';
+	source_identifier: string;
 	source: string;
```

Comment:
Should we list here possible options?

Reply:
Oh, that would be amazing! Our source is either the card brand or the type of payment, so that would be: `amex`, `diners`, `discover`, `jcb`, `mastercard`, `unionpay`, `visa`, `link`, `au_becs_debit`, `bancontact`, `eps`, `giropay`, `ideal`, `p24`, `sofort`, `sepa_debit`. Right?

Reply:
See https://stripe.com/docs/api/charges/object#charge_object-payment_method_details-type.
There `card`, `card_present`, `interac_present` we replace with card brands (as you list them in your list) + we support `link` ofcourse.

Reply:
We have an ENUM in the corresponding column in DB that would also require updating to keep things truly in sync. However, I think these are too many options (above). And the list may change sooner than later. How about we keep things more flexible here by dropping the use of enums?

Reply:
The source column in DB is `varchar(20)`, and their enums are not needed exactly because of how often we add new values here.

> How about we keep things more flexible here by dropping the use of enums?

@anu-rock, can you elaborate a little bit more?

Reply:
I'm sorry, I think I mixed channel and source. Please ignore my last comment :)